pkg/dicomio/reader.go

```diff
@@ -231,3 +231,14 @@ func (r *Reader) Peek(n int) ([]byte, error) {
 func (r *Reader) ByteOrder() binary.ByteOrder {
 	return r.bo
 }
+
+// PeekAtMost peeks up to n bytes from the buffer without consuming them.
+// If fewer bytes are available, it returns only the available ones without error.
+// Errors other than io.EOF and bufio.ErrBufferFull are propagated.
```

Comment:
Minor nitpick rewording suggestion:

```suggestion
// PeekAtMost peeks up to n bytes from the buffer without consuming them.
// If the underlying reader has fewer than n bytes available to read, it returns 
// only the available bytes without error.
// Errors other than io.EOF and bufio.ErrBufferFull are propagated.
```